src/templates/en/base.html

```diff
@@ -24,6 +24,11 @@
   You’re viewing a previous edition of the Web Almanac.
   Check out the <a href="/{{ lang }}/{{ DEFAULT_YEAR }}/{% if chapter_lang_exists(lang, DEFAULT_YEAR, slug) or page_lang_exists(lang, DEFAULT_YEAR, slug) %}{{ slug }}{% endif %}">{{ DEFAULT_YEAR }} edition!</a>
 </p>
+{%- elif year == DEFAULT_YEAR -%}
+<p lang="en">
+  📣 <a href="https://github.com/HTTPArchive/almanac.httparchive.org/discussions/4062">
```

Comment:
Use the block suggested above.

```suggestion
  📣 <a href="{{ self.call_for_contributors() }}">
```